src/ocean/time/timeout/TimeoutManager.d

```diff
@@ -228,6 +227,37 @@ abstract class TimeoutManagerBase : ITimeoutManager
 
         /***********************************************************************
 
+            Same as `unregister` but also removes expirty from list of currently
```

Comment:
I'd make this more specific:

"
Unregisters the specified expiry. If the expiry is present in the list of expired registrations being currently iterated over by `checkTimeouts`, then it will be removed (its `timeout` method will not be called). (This means that `drop` can be called from `timeout` callbacks, unlike `unregister`.)

If the specified expiry is currently not registered, nothing is done.
"

I'm not sure if this part of the doc for `unregister` is also relevant for `drop`:
>The subclass may reset the client instance to null after it has called this method (even if it throws an exception).

Reply:
> I'm not sure if this part of the doc

I have no idea what it does even mean

Reply:
Me neither. Maybe leave it off ;)